pkgs/applications/editors/micro/default.nix

```diff
@@ -1,5 +1,21 @@
-{ lib, buildGoModule, fetchFromGitHub, installShellFiles, callPackage }:
-
+{ lib
+, stdenv
+, buildGoModule
+, fetchFromGitHub
+, installShellFiles
+, callPackage
+, wl-clipboard
+, xclip
+, makeWrapper
+, disableXclip ? false
```

Comment:
can we flip to use `enable` instead of `disable` ? Cant find anything in docs, but its predominant word in nixos options

also other packages use words such as `support`,  `with`